### PR TITLE
RENAME INDEX doc

### PR DESCRIPTION
**Pull request by sploiselle**
Documents `RENAME INDEX`.

---
This change is [Reviewable](https://reviewable.io/reviews/cockroachdb/docs/679)

### CODE REVIEW

**Comment by sploiselle** [comment]
CircleCI passes for this locally. The broken links in the report seem like a hiccup in availability.

File: rename-index.md [hunk 1]
```diff
@@ -20,6 +20,45 @@ The user must have the `CREATE` [privilege](privileges.html) on the table.
 
 | Parameter | Description |
 |-----------|-------------|
-|  |  |
+| `IF EXISTS` | Rename the column only if a column of `current_name` exists; if one does not exist, do not return an error. |
```
Comment (jseldess) [inline rename-index.md]: I don't see the `current_name` parameter in the diagram. Should this be `index_name`?

File: rename-index.md [hunk 2]
```diff
@@ -20,6 +20,45 @@ The user must have the `CREATE` [privilege](privileges.html) on the table.
...
+| `table_name` | The name of the table with the index you want to use |
```
Comment (jseldess) [inline rename-index.md]: Period missing.

File: rename-index.md [hunk 3]
```diff
@@ -20,6 +20,45 @@ The user must have the `CREATE` [privilege](privileges.html) on the table.
...
+| `index_name` | The current name of the index |
```
Comment (jseldess) [inline rename-index.md]: Period missing.

**Pull request merged**